style.css

```diff
@@ -151,168 +186,150 @@ ul {
   top: 0;
 }
 
-section {
-  padding: 0;
-  height: 100%;
-}
-
 .grid {
   display: grid;
-  grid-gap: 10px;
-  height: 90%;
-  grid-auto-rows: 95%;
+  grid-template-columns: 1fr;
+  grid-template-rows: 90vh auto auto auto;
+}
+
+/* intro */
+h1 {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: var(--txtFieldColor);
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+h5 {
+  font-weight: 500;
+  color: var(--purple2);
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  text-transform: uppercase;
+  margin-bottom: 10px;
+}
+
+.paragraph {
+  margin: 12px 0 10px 0;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
 }
```

Comment:
- [ ] I think you should consider increasing the size to 20 px and ensure that the paragraph covers at least 60% of the body width. This will help us to ensure that the text does not overlap the parent. In addition, this is to ensure we stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) ☺️.

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/229753251-f6955ea1-2c67-48e5-aa74-0ca6521dacac.png)

Figma design 😎
![image](https://user-images.githubusercontent.com/99394374/229753372-92773808-8e54-43d9-85ba-218d4db1d640.png)

Reply:
I stick to the design as much as possible Sir about font, colors, etc.